### DISCUSSION TITLE
[Feature] Proper focus on the merge faces dialog

### DISCUSSION THREAD

**Discussion by eygraber** [immich-app/immich#5400] [Feature Request]
### The feature

When I'm merging a face from the people page (but probably applies elsewhere) and the merge confirmation dialog (sorry not sure what it's actually called) shows, I'd like to be able to use tab navigation to select between the "NO", "YES", and close selections.

However the focus isn't set correctly, so I have to cycle through a bunch of other elements on the screen until it gets there.

I think the first tab press should go to either "YES" or "NO".

Also I tried using ESC to close the dialog, but it closes the whole screen.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by jrasm91** [comment]
Implemented in #5486
